__tests__/app.test.js

```diff
@@ -61,6 +61,107 @@ describe('GET api/articles', () => {
               });
         })
     })
+     test('200 - returns articles with sorted_by=any valid column ', () => {
+       return request(app)
+        .get('/api/articles?sort_by=author')
+        .expect(200)
+        .then(({ body: { articles }}) => {
+            expect(articles).toBeSortedBy('author', {
+                descending: true
+            });
+        })
+     })
+     test('400 - when provided invalid sort_by keyword', () => {
+        return request(app)
+         .get('/api/articles?sort_by=bananas')
+         .expect(400)
+         .then(({ body: { msg }}) => {
+             expect(msg).toBe('bad request');
+         })
+      })
+      test('200 - returns articles with order_by=DESC as default ', () => {
+        return request(app)
+         .get('/api/articles')
+         .expect(200)
+         .then(({ body: { articles }}) => {
+             expect(articles).toBeSortedBy('created_at', {
+                 descending: true
+             });
+         })
+      })
+
+      test('200 - returns articles with order_by=DESC/ASC', () => {
+        return request(app)
+         .get('/api/articles?order=asc')
+         .expect(200)
+         .then(({ body: { articles }}) => {
+             expect(articles).toBeSortedBy('created_at', {
+                 descending: false
+             });
+         })
+      })
+
+      test('400 - when provided invalid order keyword', () => {
+        return request(app)
+         .get('/api/articles?order=123dsa')
+         .expect(400)
+         .then(({ body: { msg }}) => {
+             expect(msg).toBe('bad request');
+         })
+      })
+
+      test('200 - returns articles with topic=any valid topic', () => {
+        return request(app)
+         .get('/api/articles?topic=cats')
+         .expect(200)
+         .then(({ body: { articles }}) => {
+             expect(articles).toEqual([
```

Comment:
this test is quite brittle, I would still use .toMatchObject or .objectContaining here even if you're only expecting 1 article to make sure the test will withstand changes to the database. I'd rather you just be checking that all the articles in the array have the topic of cats